CO-3342 add note that unsubscribe events are optional (#9068)
braze-inc/braze-docs: _docs/_user_guide/personalization_and_dynamic_content/catalogs/catalog_triggers/price_drop_notifications.md

    <br> ![Catalog settings drawer.][2]{: style="max-width:70%;"}
    - **Fallback Catalog:** The catalog used for the subscription if there isn't a `catalog_name` property in the custom event.
    - **Custom event for subscribing:** The Braze custom event used to subscribe a user for catalog notifications. When this event occurs, the user who performed the event will be subscribed.

>     - **Custom event for unsubscribing:** The Braze custom event used to unsubscribe a user from notifications. This event is optional. If the user does not perform this event, they will be unsubscribed after 90 days or when the price drop event triggers, whichever happens first.

Reviewer: ```suggestion
    - **Custom event for unsubscribing:** The Braze custom event used to unsubscribe a user from notifications. This event is optional. If the user doesn't perform this event, they'll be unsubscribed after 90 days or when the price drop event triggers, whichever occurs first.
```
